### PR TITLE
Removing all default passwords from README files

### PR DESCRIPTION
**Pull request by DukeDavis12**
Removing all default passwords from README files.

Signed-off-by: Davis Benny <davis.benny@intel.com>

### CODE REVIEW

**Comment by ubrundax** [comment]
The changes done with this PR were reviewed successfully.

**Review by KiranSukhavasi — DISMISSED** [review]

File: component-samples/demo/aio/README.md [hunk 1]
```diff
@@ -267,9 +265,9 @@ Refer to the section [Docker Commands](../README.md/#docker-commands) / [Podman
...
 
-The PKCS12 keystore file \<fdo-pri-src\>/component-samples/demo/owner/owner_keystore.p12 contains the default Owner keys. It contains 3 PrivateKeyEntry with algorithm types: EC-256, EC-384 and RSA-2048, and should continue to hold PrivateKeyEntry with different algorithms. To insert/replace an existing PrivateKeyEntry of any particular algorithm, refer to section [Inserting Keys into Keystore](../README.md/#inserting-keys-into-keystore) to insert new certificate/private-key pair into \<fdo-pri-src\>/component-samples/demo/owner/owner_keystore.p12.
+The PKCS12 keystore files \<fdo-pri-src\>/component-samples/demo/owner/owner_keystore.p12 and \<fdo-pri-src\>/component-samples/demo/owner/manufacturer_keystore.p12 contain the default Owner & Manufacturer keys. These keystores contain 3 PrivateKeyEntry with algorithm types: EC-256, EC-384 and RSA-2048, and should continue to hold PrivateKeyEntry with different algorithms. The default keystore.p12 files are generated using the key generation script keys_gen.sh and you can [read more](https://github.com/secure-device-onboard/pri-fidoiot/tree/master/component-samples/demo#preparing-credentials-for-components) about it here.
```
Comment (prateekcr) [inline component-samples/demo/aio/README.md:270]: \<fdo-pri-src\>/component-samples/demo/owner/manufacturer_keystore.p12 ->  \<fdo-pri-src\>/component-samples/demo/manufacturer/manufacturer_keystore.p12

File: component-samples/demo/owner/README.md [hunk 2]
```diff
@@ -68,13 +68,13 @@ Owner runtime arguments:
 
   Path to the Owner keystore file containing the owner's keys.
 
-  Default value: ./owner_keystore.p12
+  The owner keystore file and path value is generated using keys_gen.sh script and is stored in creds.env file. [Read more](https://github.com/secure-device-onboard/pri-fidoiot/tree/master/component-samples/demo#preparing-credentials-for-components) about the key generation script here.
```
Comment (prateekcr) [inline component-samples/demo/owner/README.md:71]: For uniformity with other component's same section, remove word 'owner' here and at property owner_ssl_keystore

File: component-samples/demo/aio/README.md (hunk 1 above)
Comment (DukeDavis12) [reply component-samples/demo/aio/README.md:270]: Done.

File: component-samples/demo/owner/README.md (hunk 2 above)
Comment (DukeDavis12) [reply component-samples/demo/owner/README.md:71]: Done.

**Review by KiranSukhavasi — APPROVED** [review]

**Review by prateekcr — APPROVED** [review]

**Pull request merged**